ui/angular.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Did you need to upgrade Angular as part of this change? If that's a pre-requisite, would be nice to split out this task and have someone tackle it independent of the React stuff.

Reply:
Yeah, so angular doesn't pick up TSX files in version 5 of the CLI, so it needs to be upgraded to support that. Agreed for splitting it out if we can.